### ISSUE TITLE
Implement and submit a todobackend

### ISSUE THREAD

**Issue by rofr** [DevrexLabs/memstate#112]
This is a good exercise for learning and presence on the https://www.todobackend.com/ site will add to our online presence.

**Comment by Gulin7** [comment]
Hi, @rofr , could I take this issue?

**Comment by rofr** [comment]
yes, awesome! This is something you can do in a separate repo as its just a demoapplikation

**Comment by reshiattah** [comment]
Hello, @rofr . May I take on this issue? I'm eager to work on it and would appreciate any additional information or guidance you can provide. Thank you!
